packages/leancode_hooks/lib/src/use_synced_text_editing_controller.dart

```diff
@@ -1,6 +1,8 @@
 import 'package:flutter/widgets.dart';
 import 'package:flutter_hooks/flutter_hooks.dart';
 
+/// A wrapper around [useTextEditingController] that makes it easier to add
+/// [onChanged] callback to [TextEditingController].
 TextEditingController useSyncedTextEditingController(
```

Comment:
We now have 3 different hooks related to `TextEditingController` do we really need all of them?

Reply:
3? I think 2: `useDeclatativeTextEditingController` and `useSyncedTextEditingController`

Reply:
and ideally these 2 would be merged into 1

Reply:
I also counted `useTextEditingController`